### DISCUSSION TITLE
Getting started on a new project - Should I use Textual or Rich to create my TUI?

### DISCUSSION THREAD

**Discussion by jonmatthis** [Textualize/textual#19] [General]
I'm getting started on a project and I plan to incorporate a Rich-style TUI (like this one - https://www.willmcgugan.com/blog/tech/post/building-rich-terminal-dashboards/)

Does it make more sense for me to build it in Rich and move to Textual when this project is more mature?

Or is Textual mature enough that I should start with this? 

I'm ok with Textual being incomplete and in development as long as it is more-or-less stable!

**Comment by willmcgugan** [comment]
Textual isn't ready for prime time yet. I'd stick with Rich for now. You will be able to use the same content in Textual when its a bit more mature.

**Reply by jonmatthis** [reply]
That was my intuition, thanks for confirming!

And thanks for the work you're doing on these two projects! Aesthetics are SO important, and so often neglected!
